tests/pyfunc/docker/test_docker_flavors.py

```diff
@@ -0,0 +1,407 @@
+"""
```

Comment:
Is this something we should run nightly?

Reply:
Yes! As suggested by Ben, I created a follow-up JIRA for this sprint to run this periodically like cross version tests.